simulator.py

```diff
@@ -669,20 +669,20 @@ def at(self, location, city, duration):
 
                 # cold_and_flu_transmission(self, h)
                 if self.cold_timestamp is not None or h.cold_timestamp is not None:
-                    infector, infectee = h, self
+                    cold_infector, cold_infectee = h, self
```

Comment:
I'm not sure how these variables are reused anywhere in the function now; the change in variable name is not reflected in line 692 below

Reply:
infectee is being overwritten by these variables. `infectee` is being passed to the `logs`, so it is not logging the `covid infectee`

Reply:
Line #643 is where I had earlier defined `infectee`